src/fty_metric_compute.conf.in

```diff
@@ -1,3 +0,0 @@
-# create runtime directories for bios-agent-cm
```

Comment:
Why is this removed? The .in should stay, and the generated file (above) should be dropped. And also note the paths (legacy or new)?